.github/actions/configure_dbt_environment/action.yaml

```diff
@@ -5,7 +5,17 @@ runs:
   steps:
     - name: Configure dbt environment
       run: |
-        if [[ $GITHUB_REF_NAME == 'master' ]]; then
+        # GITHUB_HEAD_REF is only set on pull_request events, so if it's
+        # present, we must be in a PR context. Use the +x param expansion
+        # to distinguish empty strings: https://stackoverflow.com/a/13864829
+        if [ ! -z ${GITHUB_HEAD_REF+x} ]; then
```

Comment:
**nitpick (non-blocking):** Probably simpler to do: `if [ -n "$GITHUB_HEAD_REF" ]; then`

Reply:
Sure thing, done in 310645e5c0237b82492c09592ec27cf8a5206f83! Why does bash have [so many options](https://stackoverflow.com/a/3870055) for such a simple operation 🙃